Add Hidden Valley Overhauled data

IllusiveMan196 opened the pull request:
Add cleaning data and location

IllusiveMan196 committed:
Add Hidden Valley Overhauled data

Add cleaning data and location

pStyl3 commented on masterlist.yaml:
```diff
@@ -7889,3 +7889,14 @@ plugins:
+        crc: 0xA2898CFC
+        util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/347003)'
+        itm: 13
+    clean:
+      - crc: 0x2EF0FD91
+        util: 'FNVEdit v4.0.4'
```
We usually don't add `clean` data, if the plugin is `dirty`, so this can be removed.

IllusiveMan196 replied:
Clean data removed.

pStyl3 commented on masterlist.yaml:
```diff
@@ -7889,3 +7889,14 @@ plugins:
+    dirty:
+      - <<: *quickClean
+        crc: 0xA2898CFC
+        util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/347003)'
```
It would be ideal to use the latest version of xEdit available at the time of providing such data.
Currently that is xEdit v4.1.5k, which can be downloaded from xEdit's Discord (from within the `xedit-builds` channel).

IllusiveMan196 replied:
CRC is indeed the same in both versions, only change is absence of NexusMods link in 4.1.5k output.

pStyl3 commented on masterlist.yaml:
```diff
@@ -7889,3 +7889,14 @@ plugins:
         util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/34703)'
         itm: 1
+
+  - name: 'HiddenValleyOverhauled.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/70859/' ]
```
While this is a valid method to add url's to plugin entries, please write it like this:
```yaml
  - name: 'HiddenValleyOverhauled.esm'
    url:
      - link: 'https://www.nexusmods.com/newvegas/mods/70859/'
        name: 'Hidden Valley Overhauled'
```

IllusiveMan196 replied:
Done

IllusiveMan196 committed:
Update

New xEdit doesn't output link to its Nexus page and older FNVEdit page is dead.

IllusiveMan196 committed:
Update

Previous version copied from one of earlier entries, valid by pStyl3's words. Changed at their request. Mod uses one plugin as of now (earlier had two separate versions).

pStyl3 commented:
Thanks!
